### ISSUE TITLE
🐛 Pet commands on Virelai-charmed pets as BRD/BST

### ISSUE THREAD

**Issue by TracentEden** [DerpyProjectGroup/topaz#3181]
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my issue will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated.

## Branch affected by issue 

`base`

## Steps to reproduce

Charm a mob as 75 BRD/BST with Maiden's Virelai then use a pet command

## Expected behavior

Should not be able to use pet commands
(for more discussion see the original Eden [issue](https://github.com/EdenServer/community/issues/4956) that inspired testing on LSB)

**Comment by TeoTwawki** [comment]
~~incorrect, I've seen this done on retail.~~

~~You can absolutely charm with virelai then use commands form your subjob on the mobs you charm. You only get basics, but it works.~~

> https://ffxiclopedia.fandom.com/wiki/Maiden%27s_Virelai?oldid=63770
> 
> 2006 comment on wiki... "It does not give the same amount of control over the mob as a Beastmaster's Charm does. Charmed enemy will attack anything that attacks the caster."

Regarding what the wiki's say: when it tells you virelai doesn't give you control, that is true, because the control isn't natively available or from virelai, bard has no pet commands. That is not saying anything more than that.  I've legit seen brd/bst demonstrate controlling a pet this way, so unless some super-duper hacking was happening solely to convince me of this for "just because" reasons, those pets commands should actually work. Regardless, we'll be in retail soon and actually try it and show the results.

Related, should prolly also check: pet commands from wrong pet jobs. Smn stuff on bst, drg etc.

**Comment by claywar** [comment]
I can confirm as a former BRD (and BST) that it was in the 75 era possible at least.  Easy to confirm for later commands given this.

**Comment by TracentEden** [comment]
Ah, alright, yeah maybe I misunderstood as I thought it was confirmed not to work on retail (based on the eden issue), but yeah I do not have brd leveled on retail so not possible for me to personally check, glad someone can check

**Comment by WinterSolstice8** [comment]
![image](https://user-images.githubusercontent.com/60417494/201499133-677ba3c9-fb7d-42b9-9f76-3922e9b842e3.png)
Leveled BST on my mule's bard on retail
looks like it's just Sic, at least on this bee. will go to other higher level zones to see if there's more

**Comment by WinterSolstice8** [comment]
Still just Sic on a flamingo in sky

**Comment by TeoTwawki** [comment]
:heart: TY Winter

**Comment by WinterSolstice8** [comment]
![image](https://user-images.githubusercontent.com/60417494/201499372-6bb5ac52-22aa-4ccb-9f35-c48534e31a66.png)
Sic no longer works, apparently. it just tells me that I can't perform that action. When I tried to use regular charm from /BST sub (BST main level = 99) Sic functioned

**Comment by TeoTwawki** [comment]
of all the BS things for SE to waste dev time on, fixing a bug (I guess if they changed it, it must have been a bug?) nobody would ever want to use because virelai is so short.

..Realistically there is only one situation you should use vire: the mob you are facing has shadows but you do not want to aoe with diaga. Virelai has the bonus of removing them all (this happens even if it misses, even if mob immune to charm).

**Comment by TracentEden** [comment]
Yeah, SE probably have ninja fixed it, I searched in update notes from SE but no mention
